### PR TITLE
Make 100% width search bar

### PR DESCRIPTION
**Pull request by ghost**
This PR makes search bar 100% in column
And make full width on mobile (width < 768)

# Current

original

# After

after

### CODE REVIEW

**Review by guillep2k — APPROVED** [review]

File: public/less/_repository.less [hunk 1]
```diff
@@ -1936,6 +1936,12 @@
             display: block;
         }
     }
+
+    .ui.fluid.action.input {
+        .ui.search.action.input {
+            flex: 1;
```
Comment (6543) [inline public/less/_repository.less]: @jaqra why no use "auto" instead of "1"?

Comment (jaqra) [reply public/less/_repository.less]: `flex: 1;` equals to:

- flex-grow: 1;
- flex-shrink: 1;
- flex-basis: 0;

`flex: auto;` equals to:

- flex-grow: 1;
- flex-shrink: 1;
- flex-basis: auto;

so in this case both are same but if you want i can change

Comment (6543) [reply public/less/_repository.less]: It is more redable i think ... thats all :)

**Review by mrsdizzie — APPROVED** [review]

File: public/less/_repository.less (hunk 1 above)
Comment (6543) [reply public/less/_repository.less]: @jaqra Yes would be nice for other maintainers ...

**Review by 6543 — CHANGES_REQUESTED** [review]

**Review by 6543 — APPROVED** [review]
thanks :D

**Comment by 6543** [comment]
@jaqra if you like to restart drone add a empty commit ;)

**Review by lafriks — APPROVED** [review]

**Pull request merged**